domain-ee/ee-ep-merge-app/integration/mq_endpoint.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+import json
+
+from hoppy import async_consumer, async_publisher
+from hoppy.hoppy_properties import ExchangeProperties, QueueProperties
+from pika import BasicProperties
+from src.python_src.config import EXCHANGE
+
+exchange_props = ExchangeProperties(name=EXCHANGE)
+
+
+class MqEndpoint:
+    auto_response_files = []
```

Comment:
I would strongly suggest making this an instance-level attribute in `__init__` from the get-go, because otherwise unexpected things happen when you assign to a self attribute, which `set_responses()` does. (The class-level value is shared across instances _except_ when shadowed by an instance-level assigned value)